james/types.go

```diff
@@ -226,18 +226,22 @@ func (w *WebAdminClient) RenewSession() error {
 	return <-errChan
 }
 
-func (w *WebAdminClient) renew() error {
-	var lastErr error
-	err := wait.PollUntilContextTimeout(
+func (w *WebAdminClient) renew() (err error) {
```

Comment:
No need to declare err in the return params. Just use the err returned from the wait.PollUntilContextTimeout() and return. Sometimes it confusing to track the err

Reply:
It was initially done that way, but the problem was that wait.PollUntilContextTimeout() only returns the "context deadline reached" error, but not the errors that caused it. The actual relevant errors were "lost" in the process.